lib/asciidoctor/converter/semantic-html5.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+module Asciidoctor
+# A built-in {Converter} implementation that generates HTML 5 output
+# that maximizes the use of semantic constructs.
+class Converter::SemanticHtml5Converter < Converter::Base
+  register_for 'semantic-html5', 'sem-html5'
+
+  def initialize backend, opts = {}
+    @backend = backend
+    if opts[:htmlsyntax] == 'xml'
```

Comment:
I prefer the use of ternary expressions for single variable assignments, which will likely be enforced by the linter when we set it up.

```
syntax = opts[:htmlsyntax] == 'xml' ? 'xml' : 'html'
```